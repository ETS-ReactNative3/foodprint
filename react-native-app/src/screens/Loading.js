import React from 'react';
import {
  View,
  StatusBar,
  StyleSheet,
  Image,
} from 'react-native';

const Loading = props => {
  return (
    <View style={styles.container}>
      <StatusBar barStyle="light-content" />
      <View style={styles.logoContainer}>
        <Image style={styles.logo} source={require('../images/logo.png')} />
      </View>
    </View>
  );
};

export default Loading;

const styles = StyleSheet.create({
  logo: {},
  container: {
    backgroundColor: '#ffffff',
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
});
