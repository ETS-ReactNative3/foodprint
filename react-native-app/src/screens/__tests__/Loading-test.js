import React from 'react';

import renderer from 'react-test-renderer';
import Loading from '../Loading';

// Check current component against its snapshot
test('ErrorMessage renders correctly and is the same as its previous snapshot', () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
