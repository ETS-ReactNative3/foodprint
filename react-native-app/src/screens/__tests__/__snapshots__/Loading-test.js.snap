// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders correctly and is the same as its previous snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View>
    <Image
      source={
        Object {
          "testUri": "../../../src/images/logo.png",
        }
      }
      style={Object {}}
    />
  </View>
  <ActivityIndicator
    animating={true}
    color="white"
    hidesWhenStopped={true}
    size="small"
  />
</View>
`;
